src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.State.cs

```diff
@@ -387,6 +376,9 @@ private State()
                 // See https://github.com/dotnet/project-system/issues/5386
                 DateTime lastItemsChangedAtUtc = itemsChanged && ItemTypes.Count != 0 ? DateTime.UtcNow : LastItemsChangedAtUtc;
 
+                IImmutableDictionary<string, DateTime> additionalDependentFileTimes = projectSnapshot.AdditionalDependentFileTimes;
+                DateTime lastAdditionalDependentFileTimesChangedAtUtc = getLastTimeAdditionalDependentFilesAddedOrRemoved(additionalDependentFileTimes);
```

Comment:
Remove unnecessary parameter and use capital letter for local function name.

```suggestion
                DateTime lastAdditionalDependentFileTimesChangedAtUtc = GetLastTimeAdditionalDependentFilesAddedOrRemoved();
```

Reply:
done